dev_tools/rubocop/infrastructure.yml

```diff
@@ -11,3 +11,5 @@ Style/StringLiterals:
   EnforcedStyle: double_quotes
 Style/StringLiteralsInInterpolation:
   EnforcedStyle: double_quotes
+Bundler/OrderedGems
```

Comment:
It seems a bit silly to ask this given the contents of this change, but can we separate and alphabetize the different sections of RuboCop cops (e.g., the Bundler department would move up to the top)? 🙂 When we just toss stuff at the bottom we end up with duplicate entries like what I found re: line length in Core's old config file: https://github.com/generalassembly/core/blob/b6bef5fe6eee9f18cbcfb901fcbe7d85f6d2f24b/.rubocop.yml#L25-L32

Reply:
HAHA great catch 😂 Addressed!